### PR TITLE
Myfix

### PR DESCRIPTION
**Pull request by clarsonneur**
I push documentation update for clarity.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 *.log
 *~
 *.retry
+.idea
```
Comment (ojacques) [inline .gitignore]: Please split this change. It should not be included in this pull request

File: microservice/frontend/catalog/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eshop-catalog",
-  "version": "0.0.1",
+  "version": "0.02",
```
Comment (ojacques) [inline microservice/frontend/catalog/package.json]: Hmmmm... It looks like you are missing a dot. Or? 😁

Comment (clarsonneur) [reply microservice/frontend/catalog/package.json]: ho yes. Going to fix it now!!! Ty

Comment (clarsonneur) [reply microservice/frontend/catalog/package.json]: It should fix the issue #45

**Comment by clarsonneur** [comment]
@ojacques Are you happy with that, now???

**Comment by arneluehrs** [comment]
so what about the split of code then???

**Comment by clarsonneur** [comment]
Yeah done! Happy now?

**Comment by ojacques** [comment]
Looks good to me! :+1:
